release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -74,6 +74,12 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-networking"]
 === Networking
 
+[id="ocp-4-12-rsp-workload-hints"]
+==== Disabling realtime using workload hints removes Receive Packet Steering from the cluster 
+By default, the systemd service sets a Receive Packet Steering (RPS) mask for virtual network interfaces in a cluster. The RPS mask routes interrupt requests from virtual network interfaces according to the list of reserved CPUs defined in the performance profile. With this update, if you set `spec.workloadHints.realTime` in the performance profile to `False`, the systemd service removes any RPS configurations in the cluster because RPS is typically relevant to the realtime use case only.
```

Comment:
`the systemd service removes`
when the workload hint is false - the systemd service is removed completely and does not apply the rps mask on node startup (that happens when with this configuration change)